### DISCUSSION TITLE
Where is the manual for `config.yml` that goes in ISSUE_TEMPLATE?

### DISCUSSION THREAD

**Discussion by cjbarth** [community/community#57032] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm trying to move all my issue templates to a `.github` repo for the organization. This seems harder than I would expect, and impossible via the GitHub UI, but I'm sure there is quite a bit that I'm missing that `config.yml` can handle. However, I can't find a reference on all the options for `config.yml` and the [example](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#about-yaml-syntax-for-issue-forms) provided is bad syntax per `github-issue-forms.json`.

For reference, #16383 has more about what I'm trying to accomplish.

Does anyone have the link to manual pages for `config.yml`?

**Comment by jge162** [comment]
Are you talking about the issue_template? You add to your .GitHub directory that you can edit in markdown? I am not aware of a config.yml for issues.

**Reply by cjbarth** [reply]
In the ISSUE_TEMPLATE folder, you can put a `config.yml`, as I referenced above. That `config.yml` allows configuring issue templates. I'm actually trying to move away from the markdown format and use the new YAML format, but that is tangential.

**Reply by jge162** [reply]
I understand now. config.yml I tested does not seem to work. Central to your point, Hopefully you find documentation on it.

**Reply by cjbarth** [reply]
I managed to get one additional item working in `config.yml` from modifying the example they provided, but that's it. It must not be entirely broken, but a manual would surely help a lot :)
